### ISSUE TITLE
Tracking: wiki content still to be added to the Handbook

### ISSUE THREAD

**Issue by flexibeast** [void-linux/void-docs#327]
i've now completed a low-hanging-fruit pass of wiki articles to mark articles that i'm confident are either:

* satisfactorily covered by the Handbook; and/or
* clearly out-of-scope for the Handbook (e.g. don't contain any Void-specific information beyond something like `xbps-install <package>`).

i've marked such articles by renaming them to include a trailing dash, to make them visible on the [AllPages](https://wiki.voidlinux.org/Special:AllPages) wiki page. In the cases where there's a Handbook section directly corresponding to a wiki article - which is not always the case - i've removed the article contents and replaced it with the text "As this wiki is deprecated, and will soon disappear, please refer to \<link to Handbook section\>."

i have intentionally not provided redirects from the old page name to the new page name, so that search engines etc. get 404s and reduce the ranking of the wiki in search results. (A [Google search for 'void pulseaudio'](https://www.google.com/search?q=void+pulseaudio) currently ranks the wiki above the Handbook.)

The following is a list of all articles not so marked by this process:

* Acpi
* Adb
* Android
* AppArmor
* AppImages
* Arduino
* Automatic Login to Graphical Environment
* Automount
* Autox
* Beaglebone
* Bspwm
* Cinnamon
* Compiz
* Console mouse support
* Create Git Versions of Packages
* Cross Compiler
* Cubieboard2 NAND Install Hack
* Cubieboard2 SD-Card
* Cubox
* Cuda
* Disks
* Distcc
* Dmix
* Docker
* Dolphin
* Dynamic Kernel Module Support
* Embedded Images
* FAQ
* Firewall Configuration
* Firewall Guide
* Frequently Asked Questions
* Full Disk Encryption w/Encrypted Boot
* Guides
* Hashboot
* Hexchat
* Infrastructure
* Install LVM LUKS
* Install LVM LUKS on UEFI GPT
* Install Void Linux onto a USB Stick
* Install alongside Arch Linux
* Install android
* Install on a VPS
* Installation
* Installation: UEFI
* Installation On UEFI With MDRAID, via chroot
* Installation on UEFI, via chroot
* Installation via chroot
* Installer Partitioning
* Installing alongside Arch Linux
* Known Issues
* Libc
* Libinput
* Little Helpers
* Locked out - using various system recovery options to reset the password
* Locking the root account and using sudo
* Lutris
* Lyx: the document processor
* MP4 / h.264 in Firefox
* Macbook
* Main Page
* Main Page/es
* Man page
* Manual Install with encrypted boot
* Manual install w/full disk encryption
* Manual install w/full disk encryption (on MBR)
* Manual install with ZFS root
* Manual pages
* Mc
* Midnight Commander
* Mounting filesystems as a user with udevil
* Network filesystem
* Pinebook Pro
* Post Installation
* Raspberry Pi
* Raspberry Pi (External HDD with LUKS + Berryboot)
* Rfkill
* Rootfs Images
* Rosetta stone
* SANE
* SaltStack for Void Linux
* Samba
* Steam
* USB Armory
* VirtualBox
* Void mklive
* Xfce
* Xmonad

This list looks more substantial than it probably actually is: i've very much erred on the side of caution during the marking process (hence my reference to a "low-hanging-fruit pass"). Thus, for example:

* the "Rfkill" article is probably not needed; we could instead briefly mention it in e.g. `config/network/index.md` (it's already mentioned in `bluetooth.md`);
* the "Post Installation" article is probably adequately covered by various sections in the Handbook, but this needs to be confirmed;
* the "Dmix" article might only require a mention of `dmix` in the "ALSA" section;

and so on.

i'm now opening this issue to provide a single location for tracking what still needs to be done before we can decommission the wiki, and so that members of the Void team can help out with a second pass. As people provide their thoughts about the articles in the above list, i'll add text next to each item to reflect its status.

**Comment by ericonr** [comment]
Made everything into a checkbox, so we can more easily mark what's already been dealt with. Thanks for looking into this!

**Comment by flexibeast** [comment]
@ericonr: Great, thanks. :-) We should make sure that nothing gets checked off without a short explanation beside it - i don't want people to have to read through (what will probably end up being) a long comment thread to find an explanation.

**Comment by flexibeast** [comment]
Please be careful when editing the comment containing the task list! Twice now i've done an update, only to find that GitHub has not only not included the update, it's reverted to a much earlier version. :-/ i couldn't find a way to manually revert to the penultimate revision; if only there were a VCS that allowed that. :-P

**Comment by jeffayle** [comment]
re what #177 covers:

**yes**
- Install Void Linux onto a USB Stick 
- Installation on UEFI, via chroot - #177?

**no**
- Installation On UEFI With MDRAID, via chroot - #177?

**Comment by flexibeast** [comment]
@jeffayle: Thanks, updated.

**Comment by ericonr** [comment]
Could we try to post about this on reddit and ask for help?

**Comment by flexibeast** [comment]
i'm preparing a post for /r/voidlinux about recent work on the Handbook, and the post includes a link to this issue.

**Comment by flexibeast** [comment]
@bobertlo: Might you be able to go through the list as it currently stands, and indicate whether any of the unchecked items could be checked off?

**Comment by bobertlo** [comment]
@flexibeast will do. not tonight but ill keep this in notifications

**Comment by ericonr** [comment]
Given that the wiki is (temporarily?) down, http://web.archive.org/web/20190228221339/https://wiki.voidlinux.org/Special:AllPages

**Comment by ericonr** [comment]
- http://web.archive.org/web/20191211213225/https://wiki.voidlinux.org/AppImages
- http://web.archive.org/web/20191211213222/https://wiki.voidlinux.org/Automatic_Login_to_Graphical_Environment - a bit out of scope, we already link to the agetty manpage where this information can be obtained; might make sense to mention explicitly
- http://web.archive.org/web/20191211212418/https://wiki.voidlinux.org/Beaglebone - not sure we need this one since it's supposedly broken right now
- http://web.archive.org/web/20191211213236/https://wiki.voidlinux.org/Bspwm - very out of scope
- http://web.archive.org/web/20190228221156/https://wiki.voidlinux.org/Cinnamon - 100% out of scope
- http://web.archive.org/web/20190228221336/https://wiki.voidlinux.org/Compiz - not sure, installing should be a matter of `xbps-query -Rs compiz`
- http://web.archive.org/web/20190228221510/https://wiki.voidlinux.org/Console_mouse_support - no bloody idea. I certainly had no idea such a package existed
- http://web.archive.org/web/20190228222249/https://wiki.voidlinux.org/Create_Git_Versions_of_Packages - extremely not kosher, but I guess it should be documented somewhere at some point
- http://web.archive.org/web/20191211212230/https://wiki.voidlinux.org/Cross_Compiler - not sure what exactly still needs to be included in void-packages/README.md
- http://web.archive.org/web/20190228221137/https://wiki.voidlinux.org/Cubieboard2_NAND_Install_Hack + http://web.archive.org/web/20190228221319/https://wiki.voidlinux.org/Cubieboard2_SD-Card -  will use for ARM stuff
- http://web.archive.org/web/20190228221318/https://wiki.voidlinux.org/Cubox - don't think this one should be included, author could be invited to work on an official image, of course
- http://web.archive.org/web/20190228221709/https://wiki.voidlinux.org/Cuda - potentially solvable in packaging as well
- http://web.archive.org/web/20190228221205/https://wiki.voidlinux.org/Disks + http://web.archive.org/web/20190228221355/https://wiki.voidlinux.org/Installer_Partitioning - I can try to go through and move anything that's necessary to the partioning pages
- http://web.archive.org/web/20191211213224/https://wiki.voidlinux.org/Dynamic_Kernel_Module_Support - mostly need to mention where to find the build logs
- http://web.archive.org/web/20190228221119/https://wiki.voidlinux.org/Embedded_Images - is kind of the table of contents for ARM stuff, also included in the voidlinux.org/download page
- http://web.archive.org/web/20190228221315/https://wiki.voidlinux.org/Full_Disk_Encryption_w/Encrypted_Boot (this one covers "Manual install w/ fde*") + http://web.archive.org/web/20190228221355/https://wiki.voidlinux.org/Manual_Install_with_encrypted_boot
- http://web.archive.org/web/20191211213222/http://wiki.voidlinux.org/Hashboot - all relevant information is in void-docs, this page is also incorrect on several accounts
- http://web.archive.org/web/20191224070910/https://wiki.voidlinux.org/Infrastructure - should definitely go in InfraDocs if anywhere at all / don't think it's going to be documented
- http://web.archive.org/web/20190228221125/https://wiki.voidlinux.org/Install_LVM_LUKS + http://web.archive.org/web/20190228221355/https://wiki.voidlinux.org/Install_LVM_LUKS_on_UEFI_GPT - have to think about it
- http://web.archive.org/web/20190228221354/https://wiki.voidlinux.org/Install_android + https://gist.github.com/radare/7448139 - interesting, even has comment by @Johnnynator, but is more of a packaging / deployment thing than docs
- http://web.archive.org/web/20190228221122/https://wiki.voidlinux.org/Installation + http://web.archive.org/web/20190228221121/https://wiki.voidlinux.org/Installation:_UEFI + http://web.archive.org/web/20190228221118/https://wiki.voidlinux.org/Installation_On_UEFI_With_MDRAID,_via_chroot - should be partly covered, I can go through and check
- http://web.archive.org/web/20190228221125/https://wiki.voidlinux.org/Known_Issues - bootstrapping stuff, should be issues on GitHub or guides in the README. Not sure they are relevant anymore
- http://web.archive.org/web/20190228221532/https://wiki.voidlinux.org/Libinput - out of scope, indeed
- http://web.archive.org/web/20190228221336/https://wiki.voidlinux.org/Lyx:_the_document_processor - out of scope, I think
- http://web.archive.org/web/20190228221159/https://wiki.voidlinux.org/Macbook - I can try to collect some of the relevant info from there. A lot of it is fluff.
- http://web.archive.org/web/20190228221354/https://wiki.voidlinux.org/Manual_install_with_ZFS_root - badly formatted, I prefer the current PR
- Pinebook Pro page couldn't be found in web archive :o
- http://web.archive.org/web/20190228221200/https://wiki.voidlinux.org/Post_Installation - should be covered by configuration stuff, can take a closer look
- http://web.archive.org/web/20190228221318/https://wiki.voidlinux.org/Raspberry_Pi_(External_HDD_with_LUKS_%2B_Berryboot) -  could be interesting to add
- http://web.archive.org/web/20190228221334/https://wiki.voidlinux.org/SANE - like the idea of putting it together with printing, though I think most of it isn't required
- http://web.archive.org/web/20190228221402/http://wiki.voidlinux.org/SaltStack_for_Void_Linux - should this be in Void Docs? If yes, how to use? Pinging @Vaelatern
- http://web.archive.org/web/20190228221336/https://wiki.voidlinux.org/Samba - doesn't have much, not sure how much of it is up to date. Pinging @abenson and @ahesford
- http://web.archive.org/web/20190228221202/http://wiki.voidlinux.org/Steam - being worked on
- http://web.archive.org/web/20190228221137/http://wiki.voidlinux.org/USB_Armory - goes in ARM devices
- http://web.archive.org/web/20190228221154/https://wiki.voidlinux.org/VirtualBox - just needs some few package names
- http://web.archive.org/web/20190228221446/https://wiki.voidlinux.org/Void_mklive - compltely covered in the repo
- http://web.archive.org/web/20190228221238/https://wiki.voidlinux.org/Xfce - has some screen tearing info that's more about Xorg than XFCE. That said, it's far from Void specific.The rest is unecessary
- http://web.archive.org/web/20190228222249/https://wiki.voidlinux.org/Xmonad - if you're using Xmonad, you should have an idea of how to deal with it yourself, IMO.

**Comment by ahesford** [comment]
Regarding Samba, there is nothing specific to Void and little of general value in the archived page. I see no reason to preserve it.

**Comment by bobertlo** [comment]
I think Steam is covered by 3rd party software. It takes some doing to use the official package, but the flatpak works great.

**Comment by bobertlo** [comment]
I don't see anything that jumps out as important void specific information in xfce

**Comment by flexibeast** [comment]
@void-linux/pkg-committers: Is any of the information in https://wiki.voidlinux.org/Known_Issues still relevant?

**Comment by cinerea0** [comment]
Should the [Zabbix](https://wiki.voidlinux.org/Zabbix_-) and [Zram](https://wiki.voidlinux.org/Zram_-) pages be added to the checklist? The former has some Void-specific knowledge and the latter contains instructions for setting up Zram that seem to be different from any other tutorial on the subject. If so, I'd be happy to port them myself.

**Comment by flexibeast** [comment]
@cinerea0: Thanks for the offer. :-) However, as the trailing dash in the page names indicate, i believe they shouldn't be included in the Handbook:

* the Void-specific content in the Zabbix page is basically "install these packages and enable the services", which we expect users to be able to work out for themselves, using `xbps-query` to find the packages and [the Handbook to learn how to enable services](https://docs.voidlinux.org/config/services/index.html). There are literally hundreds of packages with services in the repos, and we don't want a page for each. And at any rate, setting up Zabbix is not something needed for a functioning Void system.

* the Zram page basically isn't Void-specific, and the Handbook is not generally a place for non-Void-specific guides that happen to not be available elsewhere. (That's why, for example, my "[D-Bus: the essentials](https://github.com/flexibeast/guides/blob/master/dbus.md)" guide is hosted in my own repo, and not included in the Handbook.)

**Comment by Vaelatern** [comment]
Re: Salt

Knowing the grains on the system is good.

Listing that the config files are under /etc/salt by default, and services are normal, is good.

Is it within the purview of the handbook to list github issues relevant to Void support in salt?

**Comment by cinerea0** [comment]
Given recent discussion on IRC, I think it would be best to remove the "Create Git Versions of Packages" entry because it's out of scope for the handbook and the details within are better covered by the xbps-src manual.

**Comment by jpie726** [comment]
Another page that has not yet been migrated to The Handbook: <https://wiki.voidlinux.org/Sway_->

**Comment by ericonr** [comment]
@jpie726 That page is outdated and not in scope for the Void Handbook.

**Comment by hk0O7** [comment]
[Fail2ban](https://wiki.voidlinux.org/Fail2ban) still seems to be missing.
